cs/src/core/Index/FasterLog/FasterLog.cs

```diff
@@ -189,7 +189,7 @@ private FasterLog(FasterLogSettings logSettings, bool oldCommitManager)
             fastCommitMode = logSettings.FastCommitMode;
 
             ongoingCommitRequests = new Queue<(long, FasterLogRecoveryInfo)>();
-            commitPolicy = logSettings.CommitPolicy ?? new DefaultCommitPolicy();
+            commitPolicy = logSettings.LogCommitPolicy ?? LogCommitPolicy.Default();
```

Comment:
No need to switch here because already initialized default in FasterLogSettings

Reply:
just an additional level of protection in case user sets it to null for some reason.